velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -160,6 +160,10 @@ class PlanBuilder {
   /// Helper class to build a custom TableScanNode.
   /// Uses a planBuilder instance to get the next plan id, memory pool, and
   /// parse options.
+  ///
+  /// Uses the hive connector by default. To use other connectors, you must
+  /// specify outputType, tableHandle and assignments. Note that if all three
+  /// are specified, any other builder arguments will be ignored.
```

Comment:
/// Uses the hive connector by default. Specify outputType, tableHandle, and assignments
/// for other connectors. If these three are specified, all other builder arguments will be ignored.

Reply:
updated accordingly